src/System.Collections/ref/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
+      netstandard;
```

Comment:
there is no API specific to UAP, so netstandard ref should be fine here. This was actually a bug in this particular Configurations.props since it only had a configuration for netcoreapp.